README.md
<p align="center">
<img src="img/solvify.png" /><br>
<a href="https://github.com/Solvify/Solvify/blob/master/LICENSE" target="_blank"><img src="https://img.shields.io/github/license/Solvify/Solvify?style=for-the-badge" /></a> <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=for-the-badge" alt="PRs Welcome" /> <a href="https://github.com/Solvify/Solvify/pulls" target="_blank"><img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/solvify/solvify?style=for-the-badge" /></a> <a href="https://github.com/Solvify/Solvify/issues" target="_blank"><img alt="GitHub issues" src="https://img.shields.io/github/issues/Solvify/Solvify?style=for-the-badge" /></a> <a href="https://github.com/Solvify/Solvify/blob/master/README.md#contributors-" target="_blank"><img alt="Github All Contributors" src="https://img.shields.io/github/all-contributors/solvify/solvify?style=for-the-badge" /></a>
</p>

## Overview

This repository contains the best solutions for practice, daily, weekly, and long challenge from [`#LeetCode`](https://github.com/Solvify/Solvify/tree/master/leetcode), [`#CodeChef`](https://github.com/Solvify/Solvify/tree/master/codechef),[`#HackerRank`](https://github.com/Solvify/Solvify/tree/master/hackerrank) and [`#HackerEarth`](https://github.com/Solvify/Solvify/tree/master/hackerearth) platforms.
<br>
If you think you can solve a question from any competitive platform better than anyone and let other know your approach.
Then we welcome your PR.

## Contents

Currently, we are covering four platforms ( [`#leetcode`](https://leetcode.com/), [`#codechecf`](https://www.codechef.com/), [`#hackerrank`](https://www.hackerrank.com/), [`#hackerearth`](https://www.hackerearth.com/) ) and four language `C++`, `Java`, `JavaScript`, `Python`.
<br>
But you can Create PR with your favorite language.

## Platforms

## Contributing Guidelines

- Review code by your own before creating PR.
- Make sure code will be well formatted and indented.
- Make sure your code works before submitting it.
- Follow [this](/CONTRIBUTING.md) tutorial to add the solution in respective Readmes.

## Our Contributors ✨

<table>
      <tr>
        <td align="center" >
            <a href="https://pol-alok.github.io/portfolio/">
                <img src="https://avatars1.githubusercontent.com/u/53791094?v=4?s=100" width="100px;" alt="" style="border-radius: 50%;" />
                <br />
                <br />
                <sub><b>Ravikant Pal</b></sub>
            </a>
            <br />
            <a href="https://github.com/Solvify/Solvify/commits?author=pol-alok" title="Activity">📋</a>
        </td>
        <td align="center" >
            <a href="#">
                <img src="https://avatars1.githubusercontent.com/u/42302544?v=4?s=100" width="100px;" alt="" style="border-radius: 50%;" />
                <br />
                <br />
                <sub><b>Manjesh Kumar</b></sub>
            </a>
            <br />
            <a href="https://github.com/Solvify/Solvify/commits?author=manjesh317" title="Activity">📋</a>
        </td>
        <td align="center" >
            <a href="#">
                <img src="https://avatars1.githubusercontent.com/u/36040912?v=4?s=100" width="100px;" alt="" style="border-radius: 50%;" />
                <br />
                <br />
                <sub><b>Sachin Kumar</b></sub>
            </a>
            <br />
            <a href="https://github.com/Solvify/Solvify/commits?author=sachin200697" title="Activity">📋</a>
        </td>
        <!-- <td align="center" >
            <a href="https://tesseractcoding.github.io/">
                <img src="https://avatars1.githubusercontent.com/u/53791094?v=4?s=100" width="100px;" alt="" style="border-radius: 50%;" />
                <br />
                <br />
                <sub><b>Sikandar Kumar</b></sub>
            </a>
            <br />
            <a href="https://github.com/Solvify/Solvify/commits?author=born-2learn" title="Activity">📋</a>
        </td> -->
      </tr>
</table>

## Code of Conduct

You can find our Code of Conduct [here](/CODE_OF_CONDUCT.md).

## License

This project follows the [MIT License](/LICENSE.md).


CHANGELOG.md
### Added

- Manjesh
  - leetcode: Running Sum of 1d Array. (#1480)
- Ravikant
  - leetcode: Maximum Depth of Binary Tree. (#104)
  - leetcode: Remove Duplicates from Sorted Array. (#26)
  - leetcode: Sqrt(x). (#69)
  - leetcode: Merge Sorted Array. (#88)
  - leetcode: Move Zeroes. (#283)
  - leetcode: Check If N and Its Double Exist. (#1346)
  - leetcode: Sort Array By Parity. (#905)
  - leetcode: Duplicate Zeros. (#1089)
  - leetcode: Replace Elements with Greatest Element on Right Side. (#1299)
  - leetcode: Validate Binary Search Tree. (#98)
  - leetcode: Binary Search Tree Iterator. (#173)
  - leetcode: Valid Mountain Array. (#941)
  - leetcode: Squares of a Sorted Array. (#977)
  - leetcode: Pairs of Songs With Total Durations Divisible by 60. (#1010)
  - leetcode: The kth Factor of n. (#1492)
  - leetcode: Determine if string halves are alike. (#709)
  - leetcode: To Lower case. (#1108)
  - leetcode: Defanging an IP Address. (#1614)
  - leetcode: Maximum Nesting Depth of the Parentheses. (#1662)
  - leetcode: Check If Two String Arrays are Equivalent. (#1678)
  - leetcode: Goal Parser Interpretation. (#1684)
  - leetcode: Determine if string halves are alike. (#1704)
  - leetcode: Shuffle the Array. (#1470)
  - leetcode: Richest Customer Wealth. (#1672)
  - leetcode: Number of Good Pairs. (#1512)
  - leetcode: How Many Numbers Are Smaller Than the Current Number. (#1365)
  - leetcode: Decompress Run-Length Encoded List. (#1313)
  - leetcode: Find the Highest Altitude. (#1732)
  - leetcode: Design an Ordered Stream. (#1656)
  - leetcode: Sum of All Odd Length Subarrays. (#1588)
  - leetcode: XOR Operation in an Array. (#1486)
  - leetcode: Create Target Array in the Given Order. (#1389)
  - leetcode: Running Sum of 1d Array. (#1480)
  - leetcode: Employee Importance. (#690)
  - leetcode: Binary Search Tree to Greater Sum Tree. (#1038)
  - leetcode: Remove Duplicates from Sorted List. (#83)
  - leetcode: Remove Duplicates from Sorted List II. (#82)
  - leetcode: Reverse Linked List. (#206)
  - leetcode: Middle of the Linked List. (#876)
  - leetcode: Convert Binary Number in a Linked List to Integer. (#1290)
  - leetcode: Binary Tree Preorder Traversal. (#144)
  - leetcode: Valid Anagram. (#242)
  - leetcode: Number of Steps to Reduce a Number to Zero. (#1342)
  - leetcode: Linked List Cycle. (#141)
  - leetcode: Remove Linked List Elements. (#203)
  - leetcode: Palindrome Linked List. (#234)
  - leetcode: Merge In Between Linked Lists. (#1669)
  - leetcode: Swapping Nodes in a Linked List. (#1721)
  - codechef: Chef and String. (#CHERSTR1)
  - codechef: Chef and Card Game. (#CRDGAME)
  - codechef: First and Last. (#FLOW004)
  - codechef: Reverse the number. (#FLOW007)
  - codechef: ATM. (#HS08TEST)
  - codechef: Lapindromes. (#LAPIN)
  - codechef: Lucky four. (#LUCKFOUR)
  - codechef: Turbo sort. (#TSORT)
  - hackerearth: Repeated K times. (#RKT)
  - hackerrank: Reverse a linked list. (#RLL)
- Sachin
  - leetcode: Intersection of Two Linked Lists. (#160)
- Sikandar


utils/util.cpp
#include <iostream>
#include "Readme.h"
using namespace std;
int main() {

    Readme readme;

    // 1672

    string id = "1480";

    // Problem link
    // https://leetcode.com/problems/richest-customer-wealth/
    string plink = "https://leetcode.com/problems/running-sum-of-1d-array/";

    // Problem name
    // Richest Customer Wealth
    string pname = "Running Sum of 1d Array";


    // Solution file relative path
    // leetcode/2021/february/practice/cpp/1672-RichestCustomerWealth.cpp 
    // Note : separator  should be forward slash not backword
    string relativePath = "leetcode/2021/february/practice/java/1480-RunningSumof1dArray.java ";


    //  "leetcode", "codechef", "hackerearth", "hackerrank" 
    string platform = "leetcode";

    Data data(id, plink, pname, readme.generateSolutionLink(relativePath), platform);
    // Add new Row
    readme.addRow(data);

    return 0;
}

